server/database/data.sql

```diff
@@ -17,6 +17,17 @@ CREATE TABLE users (
 	email VARCHAR(255) NOT NULL UNIQUE 
 )
 
+
+-- Topics table
+CREATE TABLE topics (
+id SERIAL PRIMARY KEY,
+module_id INT REFERENCES modules(id)
```

Comment:
Hi Joseph, in the draft for our database table for the topics, we have columns: module_id, topic_name, description, and reference_link. Could you please add these columns to the database? Also, it would be nice if you could insert some data into the table. Thank you.